aptos-move/aptos-vm/src/aptos_vm.rs

```diff
@@ -313,14 +311,13 @@ impl AptosVM {
     fn success_transaction_cleanup(
         &self,
         resolver: &impl MoveResolverExt,
-        user_txn_change_set_ext: ChangeSetExt,
+        user_txn_change_set: VMChangeSet,
         gas_meter: &mut impl AptosGasMeter,
         txn_data: &TransactionMetadata,
         log_context: &AdapterLogSchema,
         change_set_configs: &ChangeSetConfigs,
-    ) -> Result<(VMStatus, TransactionOutputExt), VMStatus> {
-        let storage_with_changes =
-            DeltaStateView::new(resolver, user_txn_change_set_ext.write_set());
+    ) -> Result<(VMStatus, VMOutput), VMStatus> {
+        let storage_with_changes = DeltaStateView::new(resolver, user_txn_change_set.write_set());
```

Comment:
I had a deja vu.
let's make sure to rebase on top of https://github.com/aptos-labs/aptos-core/pull/8266, but we knew that :-)